_docs_integrate/data-model-overview.md

```diff
@@ -387,18 +387,21 @@ The LocalAttributeShareInfo helps to keep track of how the LocalAttribute was re
-| deletionDate   | `string`                                                                            | The point in time an upcoming deletion is planned (for the `deletionStatus` `"ToBeDeleted"` or `"ToBeDeletedByPeer"`) or a completed deletion has been notified (for `"DeletedByOwner"` or `"DeletedByPeer"`).                                                            |
+| Name           | Type                                                                                                                                          | Description                                                                                                                                                                                                                                                                                                                                                                                                         |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| deletionStatus | `"DeletionRequestSent"` \| `"DeletionRequestRejected"` \| `"ToBeDeleted"` \| `"ToBeDeletedByPeer"` \| `"DeletedByOwner"` \| `"DeletedByPeer"` | The deletion status of the peer's counterpart of the shared Attribute. For own shared Attributes it may only be set to `"DeletionRequestSent"`, `"DeletionRequestRejected"`, `"ToBeDeletedByPeer"` or `"DeletedByPeer"`, whereas for peer shared Attributes only the values `"ToBeDeleted"` and `"DeletedByOwner"` are allowed.                                                                                     |
+| deletionDate   | `string`                                                                                                                                      | The point in time<br>{::nomarkdown}<ul><li>the Request with a DeleteAttributeRequestItem was sent (for "DeletionRequestSent")</li><li>the Response with a RejectResponseItem was received (for "DeletionRequestRejected")</li><li>an upcoming deletion is planned (for "ToBeDeleted" or "ToBeDeletedByPeer")</li><li>a completed deletion has been notified (for "DeletedByOwner" or "DeletedByPeer")</li></ul>{:/} |
```

Comment:
- Would format this list, for example, like this: "The point in time..." and then "...the Request with a DeleteAttributeRequestItem was sent (for `"DeletionRequestSent"`)**.**", etc., to make it more brittier 😆 
- Format the different `deletionStatus` as you do above please, e.g. `"DeletionRequestSent"`.

Reply:
- I like it better without the "...", but I added full stops in the end of the bullet points.
- Unfortunately, the formatting doesn't work here :(

Reply:
- Okay with me 🙆‍♀️ 
- The formatting works using the notation <code>"ToBeDeleted"</code>, for example. This is needed, because the <li> ... </li> notation is actually HTML, and Markdown does not automatically format code within HTML tags.